docs/code-analysis/language-server-protocol/rosie-language-server.md

```diff
@@ -0,0 +1,178 @@
+---
+id: rosie-language-server
+title: Language Server Based Code Analysis Integration with IDEs
+sidebar_label: Rosie Language Server
+description: Integration of the Codiga Code Analysis platform with various IDEs using Codiga's Rosie Language Server.
+keywords: [
+    language server,
+    language client,
+    LSP,
+    language server protocol,
+    IDE,
+    text editor,
+    vs code
+]
+---
+
+Codiga's Rosie Language Server enables users and vendors to integrate Codiga's code analysis platform into
+text editors and IDEs that haven't had such integration yet.  
+
+The Rosie Language Server is not yet available as a separate npm package, for now it lives as a part of the Codiga [VS Code extension's GitHub repository](https://github.com/codiga/vscode-plugin),
+but as a separate Node module in the `server` directory.
+
+Please note that configuration of the language client and the server launch can differ greatly between client applications (i.e. text editors, IDEs)
+depending on in what way they provide integration with language servers.
+
```

Comment:
When we have individual pages for different clients we should have a include a section in them about the differences and a link to them here